libultraship/libultraship/Lib/Fast3D/gx2_shader_gen.c

```diff
@@ -597,7 +601,7 @@ static int generatePixelShader(GX2PixelShader *psh, struct CCFeatures *cc_featur
     psh->regs.spi_ps_input_cntls[0] = 0 | (1 << 8);
 
     // inputs
-    for (int i = 0; i < num_ps_inputs; i++) {
+    for (uint32_t i = 0; i < num_ps_inputs; i++) {
```

Comment:
```suggestion
    for (size_t i = 0; i < num_ps_inputs; i++) {
```
Shouldn't this be size_t?

Reply:
`num_ps_inputs` is a uint32_t and is used below in the shader configuration.
No need to use `size_t` here.